cypresse2e/Dockerfile.dev

```
@@ -0,0 +1,11 @@
+FROM cypress/base:12.1.0
```

Do we need a Dockerfile for dev? I don't see the advantage of having two different Dockerfiles

---

@mannykary 

I saw that Dockerfile.dev is used for other services so I added it. So may I safely delete this file?

---

Well, even if we need Dockerfile for dev env, we can make it into one - 

```suggestion
FROM node:12-alpine as builder
ARG GIT_COMMIT
ARG NODE_ENV
WORKDIR '/app'
COPY package.json package-lock.json ./
RUN npm install --production
COPY . .
RUN echo "{\"version\": \"$GIT_COMMIT\"}" > version.json

FROM builder
USER node
COPY --from=builder --chown=node:node /app .
CMD ["npm", "start"]

```
The `NODE_ENV` arg is not used anywhere. I guess it is defined to use for either `dev` or `production` env. 
We can put `dev` as a default value and if we need to run it in production then we ll pass the `production` value.

Also if we see the `docker-compose` file, there the `NODE_ENV: dev` but it is using `dockerfile: Dockerfile.dev` where there is no arg called `NODE_ENV `

---

For Dockerfile in `cypress-e2e` . 

I think we really don't need Docker.dev file. Only concern when it will be in Production  and I can see for cypress-e2e  if something can be changed it may be the URL ` CYPRESS_baseUrl=http://client:3000` . 
We can define it in a variable in `docker-compose`  file and pass it depending on the env. For the moment we can keep it like this. And you can safely remove either ` Docker.dev` or `Docker`  file  and update accordingly  the `docker-compose`

---

Hi @royki 

As per our conversation https://fightpandemics.slack.com/archives/D0170MZ0NQ7/p1595332944022300
I'll remove DockerFile.dev and update docker-compose file